components/rewards/RewardsHome.js

```diff
@@ -1,65 +1,68 @@
-import { ContentContainer, WaluigiContainer } from '../../styled/rewards';
+import { View, ScrollView } from 'react-native';
+import { Body, Overline, Title } from '../BaseComponents';
+import {
+  ProgressBarContainer,
+  RewardCardContainer
+} from '../../styled/rewards';
+import { ProgressBar } from 'react-native-paper';
+import RewardTab from './RewardTab';
+import Colors from '../../assets/Colors';
 
 /**
  * @prop
  * */
 
-function createList(N) {
+function createPairs(N) {
   let foo = [];
-
-  for (var i = 1; i <= N; i++) {
+  for (var i = 1; i <= N / 2; i++) {
     foo.push(i);
   }
   return foo;
 }
 
+function createSingle(N) {
+  let foo = [];
+  if (N % 2 == 1) {
+    foo.push(1);
+  }
+  return foo;
+}
+// TODO @kennethlien to fix infinite scroll
 function RewardsHome({ transactions, user }) {
   return (
     <View>
       <ScrollView style={{ marginTop: 45 }}>
-        <WaluigiContainer>
-          <Overline>REWARD PROGRESS</Overline>
-          <Text style={{ fontSize: 10 }}> </Text>
-          <Text
-            style={{
-              fontSize: 30
-            }}>
+        <ProgressBarContainer>
+          <Overline style={{ marginBottom: 15 }}>REWARD PROGRESS</Overline>
+          <Title style={{ marginBottom: 15 }}>
             {parseInt(user.points) % 1000} / 1000
-          </Text>
-          <Text style={{ fontSize: 3 }}> </Text>
+          </Title>
           <ProgressBar
-            style={{
-              flex: 0.8,
-              height: 20,
-              borderRadius: 20
-            }}
+            style={{ height: 20, borderRadius: 20, marginBottom: 15 }}
             progress={(parseInt(user.points) % 1000) / 1000}
-            color={Colors.green800}
+            color={Colors.primaryGreen}
           />
-          <Text style={{ fontSize: 3 }}> </Text>
-          <Body>
-            Earn{' '}
-            {`${1000 -
-              (parseInt(user.points) %
-                1000)} points to unlock your next $5 reward`}{' '}
+          <Body style={{ marginBottom: 30 }}>
+            Earn {`${1000 - (parseInt(user.points) % 1000)}`} points to unlock
+            your next $5 reward
           </Body>
-          <Body> </Body>
-          <Overline>
+          <Overline style={{ marginBottom: 5 }}>
             AVAILIBLE REWARDS ({Math.floor(parseInt(user.points) / 1000)})
           </Overline>
-        </WaluigiContainer>
-        {createList(Math.floor(parseInt(user.points) / 1000)).map(a => (
-          <RewardsCard></RewardsCard>
+        </ProgressBarContainer>
+
+        {createPairs(Math.floor(parseInt(user.points) / 1000)).map(a => (
+          <RewardCardContainer>
+            <RewardTab></RewardTab>
+            <RewardTab></RewardTab>
+          </RewardCardContainer>
+        ))}
+        {createSingle(Math.floor(parseInt(user.points) / 1000)).map(a => (
+          <RewardCardContainer>
+            <RewardTab></RewardTab>
+          </RewardCardContainer>
```

Comment:
You can also move all of these rewards cards into the container above (currently named ProgressBarContainer) since you're applying the same left and right margins anyways.